style.css

```diff
@@ -650,6 +622,136 @@ article h1 {
     outline: none;
     height: 70px;
   }
+
+  .modal {
+    background-color: #e5e5e5;
+    height: 100%;
+    width: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    justify-content: center;
+    overflow: scroll;
+  }
```

Comment:
- [ ] Dear @danutibe07, Good job adding Most of the changes, But as you can see and as requested by the previous reviewer your modal needs more alignment with the given Figma template. Specially for the margin and the padding (Spacing between elements).

### Figma design 
![](https://user-images.githubusercontent.com/58642949/217996592-93dad669-d3ac-40e5-92b0-67a39a4ce412.png)
### Your design 
![Screenshot 2023-02-10 083025](https://user-images.githubusercontent.com/69077061/218009538-371f8511-ceee-4907-a441-1b6bd0c59fd4.png)